enterprise/frontend/src/metabase-enterprise/plugins.js

```diff
@@ -5,9 +5,8 @@ import MetabaseSettings from "metabase/lib/settings";
 // NOTE: temporarily use "latest" for Enterprise Edition docs
 MetabaseSettings.docsTag = () => "latest";
 MetabaseSettings.isEnterprise = () => true;
-// PLUGINS:
 
-// import "./management";
+// PLUGINS:
```

Comment:
@ranquild should this be a part of `advanced_config`? It is not mentioned in Cam's message and the doc about new feature flags. @brunobergher is it correct that we are adding notifications & alerts management as a part of `audit-app`? It looks like this small feature does not belong to audit but seems related

Reply:
This is part of `audit-app` for two reasons:
- It's a small item on its own, but part of the bigger subscription audit UI (which hasn't merged yet)
- Cam is weary of adding things behind a new flag this close to shipping

Reply:
There is a message from @camsaul 10 hours ago about moving the BE handler behind `advanced_config` feature flag. What should I do?

Reply:
Fixed